### ISSUE TITLE
SCSI-Controller selection does not work

### ISSUE THREAD

**Issue by jtruestedt** [theforeman/foreman_fog_proxmox#143]
If you create a new proxmox VM via foreman, there are 4 different scsi-controller to select in the dropdown-menu (scsi, virtio, ...) on the "virtual machine"-tab.

If you create or edit a host via the Proxmox UI there are more than 4 options available (all of them with different names than the ones available in foreman). 

The selection I provide in the foreman virtual machine tab does not have any effect on the created VM. In all cases no entry for the scsi-controller is written in the conf-File for VM. Thus, Proxmox falls back on its default value (LSC xxx).

**To Reproduce**
1. Create different VMs via foreman, selecting different scsi-controller
2. Check in the Proxmox UI
3. Check on the Proxmox-Node in the machine-conf-files
4. Notice that the selection has no effect

**Expected behavior**
I expect to have the possibility to configure the scsi-controller to be virtio or something else than the fallback solution

The tested version was foreman 1.22 with Proxmox 6 together with the foreman-fog-proxmox 0.9.3 
However, I see no recent commits, which may have changed that.

**Comment by tristanrobert** [comment]
Try to use last release: 0.10.2

**Comment by jtruestedt** [comment]
Hi,

ok I mixed something up.
I want to select the SCSI-Controller. There is something in the foreman-GUI named SCSI-Controller but it sets the device-type in Proxmox and there is no selection of the SCSI-Controller available.

I tried it with 0.10.2 (with foreman 1.24.2) and this is what I can select for the storage:
![image](https://user-images.githubusercontent.com/29541886/78779971-31789080-799e-11ea-96a9-c29db6a2be9c.png)

what I get is a machine like that (selected virtio)
![image](https://user-images.githubusercontent.com/29541886/78780377-c5e2f300-799e-11ea-872b-a6a4d33a44c4.png)

or when I select (sata):
![image](https://user-images.githubusercontent.com/29541886/78780863-954f8900-799f-11ea-84a1-eba794815707.png)

So at the harddisk in the brackets I get sata or virtio as a device corresponding to the SCSI-Controller I set in the foreman-GUI.
However, in Proxmox there is still the SCSI-Controller to select. In both cases it falls back to LSI 53C895A
![image](https://user-images.githubusercontent.com/29541886/78781566-c9777980-79a0-11ea-944d-076149af0be9.png)

Is there any possibility to select the SCSI-Controller and not the disk type?
